components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Publisher/src/Clients/v2/PublisherJobClient.cs

```diff
@@ -220,7 +219,7 @@ public sealed class PublisherJobClient : IPublishServices<string> {
                             Name = endpointId
                         },
                         ExtensionFields = new Dictionary<string, string> {
-                            [nameof(MonitoredItemSampleModel.EndpointId)] = endpointId
+                            ["EndpointId"] = endpointId
```

Comment:
I removed the dependency on the MonitoredItemSampleModel class from Publisher. This was the only place the class was used. I believe this class belongs to the telemetry processor above IoTHub API.

Reply:
Sorry, meant removal of nameof()

Reply:
yes, I had to remove the nameof() because the MonitoredItemSampleModel  is no longer accessible within the publisher. since the MonitoredItemSampleModel  is defined now in Subscriber, I wanted to avoid a dependency on the subscriber within the publisher just because this nameof().